### PR TITLE
refactor(alarm): adjust the structure of alarm message

### PR DESCRIPTION
**Pull request by ysjemmm**
#### What type of PR is this?
type-refactor

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-server/src/main/java/com/oceanbase/odc/server/OdcServer.java [hunk 1]
```diff
@@ -89,10 +89,10 @@ public static void main(String[] args) {
             log.info("Task executor exit.");
             return;
         }
-        AlarmUtils.alarm(SERVER_RESTART, LocalDateTime.now().toString());
         initEnv();
         System.setProperty("spring.cloud.bootstrap.enabled", "true");
         PluginSpringApplication.run(OdcServer.class, args);
+        AlarmUtils.alarm(SERVER_RESTART, LocalDateTime.now().toString());
```
Comment (yhilmare) [inline server/odc-server/src/main/java/com/oceanbase/odc/server/OdcServer.java:95]: this statement may never be run?
odc seems to be hang up at `PluginSpringApplication.run(OdcServer.class, args);`

Comment (ysjemmm) [reply server/odc-server/src/main/java/com/oceanbase/odc/server/OdcServer.java:95]: will execute if start
image

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]
lgtm

**Pull request merged**